src/_shared/scss/_brandbadge.scss

```diff
@@ -0,0 +1,145 @@
+.badge {
```

Comment:
Actually the GLABS team changed `advert__badge` to `badge` last week. We can probably get rid of `advert__badge` ourselves and use this one instead.